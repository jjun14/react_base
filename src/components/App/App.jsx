import React from 'react';
import imageStore from 'images/imageStore';
import styles from './App.module.scss';

const App = () => (
  <div className={styles.App}>
    <header className={styles.header}>
      <img src={imageStore.logo} className={styles.logo} alt="logo" />
      <p>Edit src/App.js and save to reload.</p>
      <a
        className={styles.link}
        href="https://reactjs.org"
        target="_blank"
        rel="noopener noreferrer"
      >
        Learn React
      </a>
    </header>
  </div>
);

export default App;
