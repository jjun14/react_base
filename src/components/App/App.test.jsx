import React from 'react';
import App from './App';

it('renders without crashing', () => {
  expect(shallow(<App />)).toMatchSnapshot();
});
