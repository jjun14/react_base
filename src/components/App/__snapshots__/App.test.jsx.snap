// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="container"
>
  <header
    className="header"
  >
    <img
      alt="logo"
      className="logo"
      src="logo.svg"
    />
    <p>
      Edit src/App.js and save to reload.
    </p>
    <a
      className="link"
      href="https://reactjs.org"
      rel="noopener noreferrer"
      target="_blank"
    >
      Learn React
    </a>
  </header>
</div>
`;
